website/versioned_docs/version-0.20/getting-started/build-a-sample-app.mdx

```diff
@@ -46,7 +46,7 @@ To convert this simple command line application to a basic Yew web application,
 
 #### Update Cargo.toml
 
-Add `yew` to the list of dependencies.
+Add `yew` to the list of dependencies editing file:
```

Comment:
```suggestion
Add `yew` to the list of dependencies by updating your Cargo.toml:
```

Reply:
@hamza1311 suggested changes implemented in PR #3406.